pkg/minishift/openshift/registry.go

```diff
@@ -21,10 +21,53 @@ import (
 	"fmt"
 	"strings"
 
+	"github.com/minishift/minishift/pkg/minishift/addon/manager"
 	instanceState "github.com/minishift/minishift/pkg/minishift/config"
 )
 
-func GetDockerRegistryInfo() (string, error) {
+func GetDockerRegistryInfo(addOnManager *manager.AddOnManager) (string, error) {
+	var registryInfo string
+	var err error
+	if isRegistryRouteEnable(addOnManager) {
```

Comment:
I find this more correct. It won't work in the case where the route add-on is applied (opposed to enabling it and applying it via start), but for the case where we get here as part of the enabled route add-on it is correct and in this case we can assume and hard code the port (443).

I could live with this. The alternative imo is to just include the add-on and make no changes to the registry command. I am somewhat 50/50 between these two alternatives.

@minishift/minishift-dev Thoughts?